rustls/src/tls12.rs

```diff
@@ -86,8 +86,13 @@ impl Tls12CipherSuite {
 }
 
 impl Suite for Tls12CipherSuite {
-    fn handler(&self) -> &'static dyn crate::server::ServerHandler<Self> {
-        crate::server::TLS12_HANDLER
+    fn handlers(
```

Comment:
The failure of `limitedclient_does_not_contain_tls12_code` bisects to this commit.